ext/com_dotnet/com_extension.stub.php

```diff
@@ -54,32 +54,31 @@ function variant_set_type(variant $variant, int $type): void {}
 
 function variant_cast(variant $variant, int $type): variant {}
 
-function com_get_active_object(string $progid, ?int $code_page = null): variant {}
+function com_get_active_object(string $prog_id, ?int $code_page = null): variant {}
 
 function com_create_guid(): string|false {}
 
-function com_event_sink(variant $comobject, object $sinkobject, array|string|null $sinkinterface = null): bool {}
+function com_event_sink(variant $variant, object $sink_object, array|string|null $sink_interface = null): bool {}
 
-/** @param com|dotnet|variant|string $comobject */
-function com_print_typeinfo($comobject, ?string $dispinterface = null, bool $wantsink = false): bool {}
+function com_print_typeinfo(variant|string $variant, ?string $disp_interface = null, bool $want_sink = false): bool {}
 
 function com_message_pump(int $timeoutms = 0): bool {}
```

Comment:
`$timeout_milliseconds` or just `$timeout` would be in line with other similar params

Reply:
I went with the more verbose `$timeout_milliseconds`, because I don't think this parameter will be used as named parameter.